app-server/src/com/thoughtworks/go/server/util/ServletHelper.java

```diff
@@ -25,13 +25,9 @@
 
     private static ServletHelper instance;
 
-    public static void init(Boolean usingJetty9) {
+    public static void init() {
         try {
-            if (usingJetty9) {
-                instance = getAppServerHelper("com.thoughtworks.go.server.util.Jetty9ServletHelper");
-            } else {
-                instance = getAppServerHelper("com.thoughtworks.go.server.util.Jetty6ServletHelper");
-            }
+            instance = getAppServerHelper("com.thoughtworks.go.server.util.Jetty9ServletHelper");
```

Comment:
Do you think we can simplify this by just instantiating the class, instead of all reflection magic that is going on?